erpnext/patches/v7_0/rename_prevdoc_fields.py

```diff
@@ -0,0 +1,22 @@
+import frappe
+from frappe.model.utils.rename_field import rename_field
+
+def execute():
+	frappe.reload_doctype('Purchase Order Item')
+	frappe.reload_doctype('Purchase Receipt Item')
+	update_po_fields()
+	rename_pr_fields()
+
+def update_po_fields():
+	for data in frappe.db.sql(""" select prevdoc_docname, prevdoc_detail_docname, name, prevdoc_doctype
+		from `tabPurchase Order Item` where prevdoc_doctype is not null""", as_dict=True):
+		if data.prevdoc_doctype == 'Material Request':
+			frappe.db.set_value("Purchase Order Item", data.name, "material_request", data.prevdoc_docname)
```

Comment:
Add `update_modified=False`

We should not change modified date of any documents.